test/known_issues/test-cluster-import-scheduling-policy.mjs

```diff
@@ -0,0 +1,13 @@
+// Refs: https://github.com/nodejs/node/issues/49240
+// When importing cluster in ESM, cluster.schedulingPolicy cannot be set;
+// and the env variable doesn't work since imports are hoisted to the top.
+// Therefore the scheduling policy is still cluster.SCHED_RR.
+import '../common/index.mjs';
+import assert from 'node:assert';
+import * as cluster from 'cluster';
+
+
+assert.strictEqual(cluster.schedulingPolicy, cluster.SCHED_RR);
+cluster.setupPrimary({ schedulingPolicy: cluster.SCHED_NONE });
+const settings = cluster.getSettings();
```

Comment:
I could be mistaken, but since `getSettings()` is an internal API, does this test need the `--expose-internals` flag?

Reply:
If this *doesn't* need `--expose-internals`, then I imagine we're exposing `getSettings()` publicly unintentionally......

Reply:
Actually I'm afraid getSettings() is being made public. I guess I would need to do some wizardry in lib/cluster.js to make it really internal 🤔 Should I do it?

Reply:
`lib/cluster.js` imports `lib/internal/cluster/primary.js` and re-exports it. So the options would seem to be to either re-engineer `lib/cluster.js` so it re-exports a property and not the whole module--shouldn't be too hard to do but you'll need to do it in a few places because it treats `lib/internal/child.js` the same way--or to find another place to export `getSettings()` from (such as `utils.js` but that doesn't import `primary` currently). I think the first option is the way to go, but other people may have more informed opinions.

Reply:
Why not export `getSettings()` from `cluster` itself? And document it, marking [`cluster.settings`](https://nodejs.org/api/cluster.html#clustersettings) as deprecated. Exporting attributes is not a good practice in the age of `import`s anyway.